VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.ts

```diff
@@ -100,7 +100,8 @@ export class JudgeContextMenuComponent {
     }
 
     private showCaseRole() {
-        return this.participant.caseTypeGroup.toLowerCase() === CaseTypeGroup.NONE.toLowerCase() ||
+        return !this.participant.caseTypeGroup ||
```

Comment:
caseTypeGroup can be a boolean? Or are we checking to see if its not null and then use toLowerCase here?

Reply:
This is just a null check.